src/ansys/dpf/gatebin/_version.py

```diff
@@ -1,6 +1,6 @@
 """Version for ansys-dpf-gatebin"""
 # major, minor, patch
-version_info = 0, 4, "2.dev0"
+version_info = 0, 5, "0.dev0"
```

Comment:
```suggestion
version_info = 0, 4, "2.dev0"
```
This will be modified automatically be our pipelines upon switch to a 242 server